cloudinit/config/cc_ssh.py

```diff
@@ -9,43 +9,26 @@
 """
 SSH
 ---
-**Summary:** configure ssh and ssh keys
+**Summary:** configure ssh and ssh keys (host and authorized)
 
-This module handles most configuration for ssh and ssh keys. Many images have
-default ssh keys, which can be removed using ``ssh_deletekeys``. Since removing
-default keys is usually the desired behavior this option is enabled by default.
+This module handles most configuration for ssh and both host and authorized ssh
+keys.
 
-Keys can be added using the ``ssh_keys`` configuration key. The argument to
-this config key should be a dictionary entries for the public and private keys
-of each desired key type. Entries in the ``ssh_keys`` config dict should
-have keys in the format ``<key type>_private`` and ``<key type>_public``, e.g.
-``rsa_private: <key>`` and ``rsa_public: <key>``. See below for supported key
-types. Not all key types have to be specified, ones left unspecified will not
-be used. If this config option is used, then no keys will be generated.
+Authorized Keys
+^^^^^^^^^^^^^^^
 
-.. note::
-    when specifying private keys in cloud-config, care should be taken to
-    ensure that the communication between the data source and the instance is
-    secure
-
-.. note::
-    to specify multiline private keys, use yaml multiline syntax
+Authorized keys are a list of public SSH key pairs that are allowed to
+connect to a system. Authorized keys for the default user/first user defined
```

Comment:
I think "first user defined in ``users``" is a touch long for a /. Perhaps a parenthetical:

```
for the default user (or the first user defined in ``users``) can be specified
```

Reply:
This expanded sentence will make it a little easier for people to key in on this being the thing they are looking for, I think:

```suggestion
connect to a user account on a system. They are stored in `.ssh/authorized_keys` in that account's home directory. Authorized keys for the default user/first user defined
```

Reply:
Done - I do think we are now in the realm of re-writting this module's docs :)